packages/vuetify/src/labs/VCalendar/VCalendar.tsx

```diff
@@ -36,6 +36,9 @@ export const VCalendar = genericComponent<VCalendarSlots>()({
   emits: {
     next: null,
     prev: null,
+    'click:monthDay': (value: Date) => true,
+    'click:weekDay': (value: Date) => true,
+    'click:day': (value: Date) => true,
```

Comment:
I guess these events should use any since we're using Date adapter and not all adapters using Date (`onXXXClick` functions should be also changed: check VDatePicker.tsx)

In fact, `onClickToday` is wrong:
```ts
    function onClickToday () {
      model.value = [new Date()]
    }
```

should be:
```ts
    function onClickToday () {
      model.value = [adapter.date()]
    }
```

I need to check if this is working:

![imagen](https://github.com/vuetifyjs/vuetify/assets/6311119/f57fcf65-ab20-4e45-8db5-412e7e57563f)